server/src/main/java/org/apache/druid/server/coordinator/duty/EmitClusterStatsAndMetrics.java

```diff
@@ -133,6 +142,21 @@ public DruidCoordinatorRuntimeParams run(DruidCoordinatorRuntimeParams params)
     CoordinatorStats stats = params.getCoordinatorStats();
     ServiceEmitter emitter = params.getEmitter();
 
+    if (DruidCoordinator.HISTORICAL_MANAGEMENT_DUTIES_DUTY_GROUP.equals(groupName)) {
+      emitStatsForHistoricalManagementDuties(cluster, stats, emitter, params);
+    }
+    if (dutyList.stream().anyMatch(duty -> duty instanceof CompactSegments)) {
```

Comment:
Instead of passing dutiesList as a parameter to the constructor, I'd recommend passing in a boolean parameter, to make it clear that this duty doesn't deal with any of the other duties. Otherwise LGTM

Reply:
Done